spec/views/hq/orphans/show_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe "hq/orphans/show.html.haml", type: :view do
+  let(:orphans) { FactoryGirl.build_stubbed(:orphan_list) }
+  let(:orphan) { FactoryGirl.build_stubbed(:orphan, orphan_list: orphans) }
+
+  describe 'the orphan exists' do
+    before :each do
+      assign(:orphan, orphan)
```

Comment:
Yes, you are right. I try to follow the same idea of `sponsor`. But for orphan is more simple.